storagetransfer/aws_request.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Command-line sample that creates a one-time transfer from Amazon S3 to
+Google Cloud Storage.
+"""
+
+
+import argparse
+
+# [START storagetransfer_transfer_from_aws]
+from datetime import datetime
+
+from google.cloud import storage_transfer
+
+
+def create_one_time_aws_transfer(project_id: str, source_bucket: str,
+                                 aws_access_key_id: str,
+                                 aws_secret_access_key: str, sink_bucket: str):
+    """Creates a one-time transfer job from Amazon S3 to Google Cloud
+    Storage."""
+
+    client = storage_transfer.StorageTransferServiceClient()
+
+    # The ID of the Google Cloud Platform Project that owns the job
+    # project_id = 'my-project-id'
+
+    # AWS S3 source bucket name
+    # source_bucket = 'my-s3-source-bucket'
+
+    # AWS Access Key ID
+    # aws_access_key_id = 'AKIA...'
+
+    # AWS Secret Access Key
+    # aws_secret_access_key = 'HEAoMK2.../...ku8'
+
+    # Google Cloud Storage destination bucket name
+    # sink_bucket = 'my-gcs-destination-bucket'
+
+    now = datetime.utcnow()
+    # Setting the same day, month, and year signifies a one-time transfer
```

Comment:
I think this comment is a little confusing. I think you mean "Setting the one start date and the end date as the same time creates a one-time transfer,"